<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomRectSceneItemEdit</class>
 <widget class="QWidget" name="CustomRectSceneItemEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomRectSceneItemEdit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Custom Rect Item AyarlarÄ±</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditRectWidth"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEditRectHeigth"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIsFilled">
        <property name="text">
         <string>is Filled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFillColor">
        <property name="text">
         <string>Fill Color</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>General Thickness:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditGeneralThickness"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>General Border Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGeneralBorderColor">
          <property name="text">
           <string>Select General Border Color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Custom Edge Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxCustomTopEdge">
             <property name="text">
              <string>Top Edge</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QCheckBox" name="checkBoxCustomLeftEdge">
             <property name="text">
              <string>Left Edge</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBoxCustomRightEdge">
             <property name="text">
              <string>Right Edge</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Thickness:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="checkBoxCustomButtomEdge">
             <property name="text">
              <string>Bottom Edge</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineEditTopEdgeThickness"/>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButtonTopEdgeColor">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditRightEdgeThickness"/>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEditBottomEdgeThickness"/>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEditLeftEdgeThickness"/>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButtonRightEdgeColor">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pushButtonBottomEdgeColor">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="pushButtonLeftEdgeColor">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonApplyChanges">
        <property name="text">
         <string>APPLY CHANGES TO RECT ITEM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
